### PR TITLE
Fix few spelling errors detected by Grammarly

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: docs/publications/BioHackrXiv/paper.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ These are described in more detail below.
 
 ## Prototype interactive table for Galaxy communities
 
-The described workflow for the Galaxy tool metadata extractor (see Figure \ref{metadata_extractor_pipeline}) was successfully implemented ([GitHub repository](https://github.com/galaxyproject/galaxy_tool_extractor)) and could more then 1,300 Galaxy tool suites (see the [GitHub repository pages](https://galaxyproject.github.io/galaxy_tool_metadata_extractor) for an up-to-date table).
+The described workflow for the Galaxy tool metadata extractor (see Figure \ref{metadata_extractor_pipeline}) was successfully implemented ([GitHub repository](https://github.com/galaxyproject/galaxy_tool_extractor)) and could more than 1,300 Galaxy tool suites (see the [GitHub repository pages](https://galaxyproject.github.io/galaxy_tool_metadata_extractor) for an up-to-date table).
```
Comment (nsoranzo) [inline docs/publications/BioHackrXiv/paper.md:179]: "could more than" is  missing something here.

File: docs/publications/BioHackrXiv/paper.md [hunk 2]
```diff
@@ -210,10 +210,10 @@ The guidelines created were also used to update the [best practices for creating
...
 The generated tabular tool summary provides valuable information that extends beyond the use case of listing community tools. Therefore, an integration with the [Research Software Ecosystem (RSEc)](https://github.com/research-software-ecosystem/content) [@citesAsAuthority:RSEc] is currently being worked on.
-Various updates of the Galaxy tool metadata extractor pipeline are also envisioned, such as the integration of comprehensive usage statistics for all large Galaxy servers, additional bio.tools metadata, and a user-friendly integration of manual curation steps.
+Various updates of the Galaxy tool metadata extractor pipeline are also envisioned, such as the integration of comprehensive usage statistics for all large Galaxy servers, and additional bio.tools metadata, and a user-friendly integration of manual curation steps.
```
Comment (nsoranzo) [inline docs/publications/BioHackrXiv/paper.md:213]: I don't think this change is correct, revert?

File: docs/publications/BioHackrXiv/paper.md (hunk 1 above)
Comment (bebatut) [inline docs/publications/BioHackrXiv/paper.md:179]:
```suggestion
The described workflow for the Galaxy tool metadata extractor (see Figure \ref{metadata_extractor_pipeline}) was successfully implemented ([GitHub repository](https://github.com/galaxyproject/galaxy_tool_extractor)) and could extract more than 1,300 Galaxy tool suites (see the [GitHub repository pages](https://galaxyproject.github.io/galaxy_tool_metadata_extractor) for an up-to-date table).
```

File: docs/publications/BioHackrXiv/paper.md (hunk 2 above)
Comment (bebatut) [inline docs/publications/BioHackrXiv/paper.md:213]:
```suggestion
Various updates of the Galaxy tool metadata extractor pipeline are also envisioned, such as the integration of comprehensive usage statistics for all large Galaxy servers, additional bio.tools metadata, and a user-friendly integration of manual curation steps.
```

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**